docs/boot-options.rst

```diff
@@ -237,9 +237,11 @@ For example:
 inst.ks.all
 ^^^^^^^^^^^
 
-All locations of type http, https or ftp specified with inst.ks will be used
-sequentially one by one until the kickstart file is fetched. Other locations
-will be ignored.
+Use all locations of type ``http``, ``https`` or ``ftp`` specified with
+multiple ``inst.ks`` sequentially one by one until kickstart file is fetched. Other
+locations will be ignored.
+
+Without this option, only first location specified by ``inst.ks`` is used.
```

Comment:
For historical reasons, it is the last location.

Reply:
thanks, fixed